### ISSUE TITLE
Tablet not displaying the geoscout as a connected device when plugged in

### ISSUE THREAD

**Issue by jordannicolette** [precision-sustainable-ag/On-farm-Protocols#295]
**What's wrong**
- When the user has the geoscout connected to their tablet to upload the data, there is no option to select the the geoscout as a connected device to view the files. The device does not show in the in their form on ODK collect or in the "files" app
- Not all users are having this issue but it has been reported to the hotline by:
-     Matthew Allen (UK)
-     Asmita Gautam + Akwasi (Purdue)
-     April Dobbs (NCSU)
-     Abby (Penn State)

There is a workaround where the user can connect their geoscout to their personal device, upload the file to google drive, then open the file on google drive on the tablet. This requires some extra steps that aren't outlined in the protocol so I want to avoid this if necessary. 

This seems to be an ongoing problem with the "files" app or within ODK, not the cord since these users have been able to connect their geoscout with another device. @saseehav @kldriver

**Comment by screberg** [comment]
I have nothing helpful to say yet.  I just need to get my head back in the game on this protocol.  Have we tried remoting into these tablets?  I don't think serial connections show up in Android's "Connected Devices" menu.  So where can we seed what is connected over serial?

**Comment by elhenriq** [comment]
I tried replicating this issue today to try and come up with a solution. I tried using a tablet that has not been provision by Esper and one that has been provision! I was able to get the geoscout to be recognized by both tablet and transfer data with no issues. 

on the provision tablet I tried with my files shown and disable and worked for both ways. there was no point in connecting with the serial port app since the way the geoscout connects to the tablet is with a USB protocol and the device is recognized as an external drive on the tablet itself. 

my next try is going to be getting a hold of April's tablet here in NC and trying to use my Geoscout box on her tablet, her geoscout box on my tablet and trying different USB cables, adapter. to try and replicate the issue.

**Comment by jordannicolette** [comment]
Awesome, let us know in here when you try it out with April's. Been having a bunch of calls this week with users having issues pulling files from the geoscout, it hasn't been coming up as a connected device so I'd like to get it sorted out

**Comment by jordannicolette** [comment]
Update 4/28: Nick Warren NH PSA - also having the same issue with the geoscout not showing up as a connected device. He is using google drive to upload his most recent scan data. @elhenriq @saseehav

**Comment by elhenriq** [comment]
@saseehav and I were able to figure out what is happening with all of these tablets. there is a setting on the Policy that those tablets are on right now that stops them from being able to connect to an external device. the policy has to be change on esper for all those tablets. 

@saseehav how should we proceed? can Jordan go ahead and update them as they contact him?

**Comment by saseehav** [comment]
Update on Tablets not connecting to GeoScout: 
[@Esleyther Henriquez](https://crownteamworkspace.slack.com/team/UAUABFBFF)
 was reviewing the differences between the tablets that connected and the ones that didn't and found that the Compliance Policy in Esper was different. We edited the Policy to include access for external USB devices. Turns out the 7 original tablets didn't have this but 45 did. As of this afternoon 20 devices have been updated (mostly CE2 devices) . In the case that you have a user who cannot connect to a USB device like the GeoScout, here are the steps to take in Esper:

Get the device online
Select the Device
In the top row tabs select "Compliance Policy"
In the drop down menu select "PSA New Update"
Apply Policy

![image](https://user-images.githubusercontent.com/49722541/235223646-a1bdccd3-f616-4a6f-b2f8-395d78e707cc.png)

@jordannicolette 
Anyone who has access to Esper can push this update. If they would like help they can have me do it. I will reach out to CE1 and On-farm teams next to ask them to put their tablets online for this reason.
